### ISSUE TITLE
Pressure stall info counters labeling is indistinctive

### ISSUE THREAD

**Issue by kkm000** [htop-dev/htop#350]
The new Pressure stall counters look valuable, thanks for adding them. They have display labeling issues, however:
1. Their names are indistinguishable in the Left/Right columns menus in the configuration screen: all 5 of them read as `Pressure Stall Infor`.
2. The labels are shortened to `Som` and `Ful` on the main screen when readouts are set to bar mode.
```
  Avg[||                                                           1.3%]   
  Mem[||||||||||||||||||||||||||||||||||||||||||||||||||||||10.2G/11.7G]   
  Som[|||                                        xxxx 0.43% 0.50% 0.46%]   
  Ful[                                           xxxx 0.00% 0.00% 0.00%]   
  Ful[                                           xxxx 0.00% 0.00% 0.00%]

Setup            Left column          Right column         Available meters
Meters           CPU [Bar]            Task counter [Text]  Clock
Display options  Memory [Bar]         Load average [Text]  Date
Colors           Pressure Stall Infor Uptime [Text]        Date and Time
Columns          Pressure Stall Infor Disk IO [Text]       Load averages: 1
                 Pressure Stall Infor                      Load: average of
                                                           Memory
```
(2) is not an issue in text readout mode:
```
  Avg[||                                                   
  Mem[|||||||||||||||||||||||||||||||||||||||||||||||||||||
  Some CPU pressure: 0.44% 0.49% 0.49%                     
  Full IO  pressure: 0.00% 0.00% 0.00%                     
  Full Mem pressure: 0.00% 0.00% 0.00%

Setup            Left column          Right column         
Meters           CPU [Bar]            Task counter [Text]  
Display options  Memory [Bar]         Load average [Text]  
Colors           Pressure Stall Infor Uptime [Text]        
Columns          Pressure Stall Infor Disk IO [Text]       
                 Pressure Stall Infor                      
```
I kinda used to the labeling of CPU load as `Avg`, but having bars labeled `Som`, `Som` and, you knew it was coming, `Som` is a different degree of inconvenience. :)

---
htop 3.0.3-dev built from the current tip of the master branch, 19b5141.
Configured as `./configure CC=clang-8 CFLAGS='-Oz'` on a Debian 10 machine with a bpo kernel 5.8.
```
$ uname -srvmo
Linux 5.8.0-0.bpo.2-amd64 #1 SMP Debian 5.8.10-1~bpo10+1 (2020-09-26) x86_64 GNU/Linux
```

**Comment by BenBE** [comment]
For the labelling issue in the setup dialog we could shorten the names to have `Pressure Stall Information` abbreviated as `PSI`, thus only showing `PSI: full CPU` as an example. This won't help with the bar display mode though, as the bar display always only shows the first 3 characters of the caption. But: We could display the exact type similar to what we do with extra information in the CPU meters and right-align the exact type there.

**Comment by fasterit** [comment]
Closing as #354 was merged and ded9c5d36332401bb8d5e33b6f72de188a42560d improves the print out
